src/Commands/Modules/SoftModerationModule.cs

```diff
@@ -8,5 +9,8 @@ namespace BrackeysBot.Commands
     [ModuleColor(0xe89797)]
     public partial class SoftModerationModule : BrackeysBotModule
     {
+
+        public EndorseService endorsements { get;set; }
```

Comment:
Public properties in C# should be uppercase. Also, why add the property to the entire module instead of just the command?

Reply:
Oh that works? I copied this from Leaderboard. Will change.